specification/compute/resource-manager/Microsoft.Compute/preview/2018-06-01/gallery.json

```diff
@@ -0,0 +1,1120 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "GalleryClient",
+    "description": "The Gallery Client.",
+    "version": "2018-06-01"
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "security": [
+    {
+      "azure_auth": [
+        "user_impersonation"
+      ]
+    }
+  ],
+  "securityDefinitions": {
+    "azure_auth": {
+      "type": "oauth2",
+      "authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
+      "flow": "implicit",
+      "description": "Azure Active Directory OAuth2 Flow",
+      "scopes": {
+        "user_impersonation": "impersonate your user account"
+      }
+    }
+  },
+  "paths": {
```

Comment:
PATCH for galleries missing. it is required for update of atleast tags and strongly recommended for supporting update of any other updatable property

Reply:
In our API spec, we use PUT to create or update the resource properties. So do we still need the PATCH?

Reply:
As per RPC, yes you do. I know in CRP PUT behaves like a PATCH, which is like a PUTCH and does not follow the http semantics of a PUT. Is there a plan to fix the behavior for new resource types?